ZUGFeRD/InvoiceDescriptor21Writer.cs

```diff
@@ -75,6 +75,14 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream)
                 Writer.WriteElementString("udt:Indicator", "true");
                 Writer.WriteEndElement(); // !ram:TestIndicator
             }
+
+            if(!String.IsNullOrEmpty(this.Descriptor.BusinessProcess))
+            {
+                Writer.WriteStartElement("ram:BusinessProcessSpecifiedDocumentContextParameter");
```

Comment:
I guess it should be profile-depended here as well?

Reply:
No, according to ZUGFeRD_1p0_Technical_Documentation.pdf Business process is Optional and only for EXTENDED profile  
but for ZUGFeRD-2.1-Specification_TA.pdf, Business process is optional and can be used in all profiles (MINIMUM, BASIC WL, BASIC, EN 16931 (COMFORT), EXTENDED)